src/components/UserInputBlock.tsx

```diff
@@ -176,8 +196,21 @@ export const UserInputBlock = ({ content, prompt, ack, waitForEvent, fireEvent,
               rows={DEFAULT_TEXTAREA_ROWS}
               className={buildClassName(responseSubmitted, errors.response)}/>
               <ErrorMessage className="text-danger my-3" component="div" name="response" />
-              <div className='os-text-center mt-4'>
-              <button type="submit" disabled={isSubmitting || responseSubmitted} className="os-btn btn-outline-primary">{buttonText}</button>
+              <div className='os-text-center mt-4 os-flex os-justify-space-evenly'>
+                <button type="submit" disabled={isSubmitting || responseSubmitted} className="os-btn btn-outline-primary">{buttonText}</button>
+                {
+                  (persistor !== undefined) &&
+                  <button
+                    type="button"
+                    onClick={(): void => {
+                      void resetPersistedState(setFieldError, setFieldValue)
+                      void setTouched({ response: true }, false)
```

Comment:
Clarification question: What's the behavioral difference if the `false` isn't passed and validation is enabled? Playing around I couldn't quite figure it out. I expected it would give a validation error without it when we reset to `''`, but that doesn't seem to be the case :man_shrugging: .

Reply:
I didn't notice a behavioral difference either, but since we were resetting to our default value I didn't see the point in validating the default value so I passed `false`. Maybe I am misunderstanding the point of validation?

Reply:
Ah, so it's just a bit of efficiency to avoid doing unnecessary work. :+1: